en_partut-ud-dev.conllu

```diff
@@ -904,7 +904,7 @@
 12	,	,	PUNCT	FF	_	11	punct	_	_
 13	this	this	PRON	PD	Number=Sing|PronType=Dem	14	nsubj	_	_
 14	is	be	VERB	V	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	8	parataxis	_	_
-15	where	where	SCONJ	CS	_	17	mark	_	_
+15	where	where	ADV	B	PronType=Rel	17	advmod	_	_
```

Comment:
If this is `PronType=Rel` there should be a relative clause. This may be interpreted an interrogative though (indirect form of the question "Where do we go to church?").